File: leetcode-tasks/3110-score-of-a-string.ts
// Accepted
// Runtime 55 ms Beats 90.71% of users with TypeScript
// Memory 51.97 MB Beats 81.97% of users with TypeScript

const scoreOfString = (s: string): number => s.split('').reduce((acc, curr, i, arr) => i ? acc + Math.abs(arr[i - 1].charCodeAt(0) - curr.charCodeAt(0)) : 0, 0)

File: tasks/3110-score-of-a-string.go
// Accepted 705 / 705 testcases passed Sergey Pomortsev submitted at Jul 03, 2026 15:22
// Runtime 0 ms Beats 100.00% Memory 3.99 MB Beats 84.21%

func scoreOfString(s string) int {
  var result float64

  for i := 0; i < len(s) - 1; i++ {
    result += math.Abs(float64(int(s[i]) - int(s[i + 1])))
  }

  return int(result)
}